nav2_docking/opennav_docking/src/dock_database.cpp

```diff
@@ -99,6 +118,17 @@ Dock * DockDatabase::findDock(const std::string & dock_id)
 
 Dock * DockDatabase::findDockInstance(const std::string & dock_id)
 {
+  if (dock_instances_.empty()) {
```

Comment:
This is not necessary, please remove. If the database is empty, `find` results in the map's `end` which returns nullptr already.

Reply:
I only add it to be more verbose for the user

Reply:
done